CHANGELOG.md

```diff
@@ -393,7 +398,7 @@ Check [Keep a Changelog](http://keepachangelog.com/) for recommendations on how
 
 - Initial release of the puppet extension.
 
-[Unreleased]: https://github.com/lingua-pupuli/puppet-vscode/compare/0.19.0...master
+[unreleased]: https://github.com/lingua-pupuli/puppet-vscode/compare/0.19.0...master
```

Comment:
Why does this keep getting changed to `unreleased` ?  The link above has capital `U`